src/store/modules/job/actions.ts

```diff
@@ -762,6 +763,137 @@ const actions: ActionTree<JobState, RootState> = {
+            'maxRecurrenceCount': '-1',
+            'parentJobId': job.parentJobId,
+            'runAsUser': 'system', //default system, but empty in run now.  TODO Need to remove this as we are using SERVICE_RUN_AS_SYSTEM, currently kept it for backward compatibility
+            'recurrenceTimeZone': this.state.user.current.userTimeZone
+          },
+          'statusId': "SERVICE_PENDING",
+          'systemJobEnumId': job.systemJobEnumId
+        } as any
+
+        
+        if (job?.runtimeData?.shopifyConfigId) {
+          const shopifyConfig = this.state.user.shopifyConfigs.find((config: any) => {
+            return config.shopId === shopId;
+          })
+
+          params['shopifyConfigId'] = shopifyConfig.shopifyConfigId;
+          params['jobFields']['shopId'] = shopId;
+        }
+
+        // checking if the runtimeData has productStoreId, and if present then adding it on root level
+        job?.runtimeData?.productStoreId?.length >= 0 && (params['productStoreId'] = payload.eComStoreId)
+        job?.priority && (params['SERVICE_PRIORITY'] = job.priority.toString())
+
+        // assigning '' (empty string) to all the runtimeData properties whose value is "null"
+        job.runtimeData && Object.keys(job.runtimeData).map((key: any) => {
+          if (job.runtimeData[key] === 'null') job.runtimeData[key] = ''
+        })
+        jobParams.push({ ...job.runtimeData, ...params });
+      }, {})
+    })
+
+    Promise.allSettled(jobParams.map(async (param: any) => {
+      const resp: any = await JobService.scheduleJob(param);
+      if(resp.status === 200 && !hasError(resp)){
+        // Removing the scheduled job
+        dispatch('removeBulkJob', param.systemJobEnumId);
+        return Promise.resolve(resp);
+      } else {
+        failedJobs++;
+        return Promise.reject(resp);
+      }
+    })).then((resps: any) => {
+      if(failedJobs > 0) {
+        return showToast(translate("Failed to schedule service(s)", {count: failedJobs}))
+      } else {
+        return showToast(translate('Services have been scheduled in bulk'))
+      }
+    })
+  },
+  setBulkJobGlobalRuntime({ commit, state }, payload) {
+    let bulkJobs = JSON.parse(JSON.stringify(state.bulk.jobs));
+    commit(types.JOB_BULK_DATA_UPDATED, { value: payload.runtime, type: 'runtime' });
+    bulkJobs = bulkJobs.map((job: any) => ({ ...job, runtime: state.bulk.runtime }));
+    commit(types.JOB_BULK_UPDATED, bulkJobs);
+  },
+  setBulkJobGlobalFrequency({ commit, state }, payload) {
+    let bulkJobs = JSON.parse(JSON.stringify(state.bulk.jobs));
+    commit(types.JOB_BULK_DATA_UPDATED, { value: payload.frequency, type: 'frequency' });
+    bulkJobs = bulkJobs.map((job: any) => {
+      if(job.freqType === 'slow') {
+        showToast(translate("Some jobs have slow frequency type, hence, feasible frequency will be set automatically"))
```

Comment:
I think this will execute for all the jobs, we should show it at last if any job is slow and we have changes the frequency for it